src/redash_stmo/settings.py

```diff
@@ -19,3 +19,13 @@
 REMOTE_GROUPS_ALLOWED = set_from_string(
     os.environ.get("REDASH_REMOTE_GROUPS_ALLOWED", "")
 )
+
+# Iodide API key for creating new notebooks
+IODIDE_API_KEY = os.environ.get(
+    "REDASH_IODIDE_API_KEY"
+)
```

Comment:
Let's call this `IODIDE_AUTH_TOKEN`/`REDASH_IODIDE_AUTH_TOKEN` since it's the technical term used on the Iodide server side.